apps/pie-docs/package.json

```diff
@@ -11,9 +11,8 @@
   "scripts": {
     "build": "DEPLOYMENT_ENVIRONMENT='production' eleventy",
     "build:dev": "DEPLOYMENT_ENVIRONMENT='development' eleventy",
-    "clean:dist": "run -T rimraf dist",
-    "cz": "run -T cz",
-    "dev": "yarn clean:dist && npx @11ty/eleventy --serve",
+    "clean": "run -T rimraf dist",
+    "dev": "npx @11ty/eleventy --serve",
```

Comment:
Out of interest - why remove the clean before serving here?

Looks like you may need to add clean as a dependency in the turbo confg for `dev` (it's added to `build:dev` atm, but not the straight up `dev` script?)

Reply:
Good question! `dev` depends on `build:dev`, which itself depends on `clean`.